apps/prairielearn/src/pages/partials/navTabs/instructorAssessment.ejs

```diff
@@ -72,3 +72,9 @@
     iconClasses:    'fas fa-upload',
     tabLabel:       'Uploads'}) %>
 <% } %>
+<%- include('navTab', {
+    activeSubPage:  'access_overrides',
+    urlSuffix:      '/assessment/'+assessment.id+'/access_overrides',
+    iconClasses:    'fa fa-undo',
+    tabLabel:       'Access Overrides'}) %>
```

Comment:
This tab should be placed immediately after the existing "Access" tab.

Reply:
Fixed in [9f36f86](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/9f36f86b3a7028e0f1709bc02fedd723d9313412)